src/libAtomVM/nifs.c

```diff
@@ -109,6 +110,8 @@ static term nifs_erlang_process_flag(Context *ctx, int argc, term argv[]);
 static term nifs_erlang_processes(Context *ctx, int argc, term argv[]);
 static term nifs_erlang_process_info(Context *ctx, int argc, term argv[]);
 static term nifs_erlang_system_info(Context *ctx, int argc, term argv[]);
+static term nifs_erlang_binary_to_term(Context *ctx, int argc, term argv[]);
```

Comment:
I am a little confused about whether this should be "nifs_" (plural) or "nif_" (singular).  Please advise on your preference, as we have two conventions in this file.  Not sure which one to use.

Reply:
I think that exported functions should be prefixed with their C filename, so all `term.c/.h` functions are prefixed with `term_`.
However those nifs.c functions are static, so they are not exported, so we can call them with any name we like. So I think that `nif_something` is a good name.

Reply:
Changes all the static functions I had prefixed `nifs_` with `nif_`.  There is still one exported function (`nifs_get`), which I think you want to keep with the module name prefix, as the function is exported.